src/main/java/io/codelex/dateandtime/practice/Exercise1/Exercise1.java

```diff
@@ -0,0 +1,43 @@
+package io.codelex.dateandtime.practice.Exercise1;
+
+import java.time.DayOfWeek;
+import java.time.LocalDate;
+
+public class Exercise1 {
+
+
+    public static void main(String[] args) {
+
+        LocalDate date1 = LocalDate.parse("2022-03-07");
+        LocalDate date2 = LocalDate.parse("2022-03-20");
+
+        // Loop to count days in general
+        int days = 0;
+        for (LocalDate date = date1; date.isBefore(date2.plusDays(1)); date = date.plusDays(1)) {
+            days++;
+        }
+
+        // Loop to count only working days.
+        int count = 0;
+        for (LocalDate date = date1; date.isBefore(date2.plusDays(1)); date = date.plusDays(1)) {
+            if (getDayNumber(date) != 6 && getDayNumber(date) != 7) {
```

Comment:
var mēģināt izmantot built in enums ar dienu nosaukumiem
un iekš if varētu izvilkt uz palīgmetodi
`if(isWorkDay())` for example

Reply:
Hey, salabots.